{% set name = "ftfy" %}
{% set version = "5.1" %}
{% set sha256sum = "2db945e2b46ef975cb9edbb897504df3bfd8f48f2be94297a3d4b7e906741ddf" %}

package:
  name: {{ name }}
  version: {{ version }}

source:
  fn: {{ name }}-{{ version }}.tar.gz
  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
  sha256: {{ sha256sum }}

build:
  number: 0
  script: python setup.py install --single-version-externally-managed --record=record.txt
  skip: True  # [py2k]
  entry_points:
    - ftfy = ftfy.cli:main

requirements:
  build:
    - python
    - setuptools

  run:
    - python
    # Required for loading package data via `pkg_resources` module.
    - setuptools
    - wcwidth

test:
  requires:
    - nose
  source_files:
    - tests
  commands:
    - PYTHONIOENCODING=utf8 ftfy tests/halibote.txt  # [unix]
    - SET "PYTHONIOENCODING=utf8" & ftfy tests/halibote.txt  # [win]

about:
  home: https://github.com/LuminosoInsight/python-ftfy/
  license: MIT
  license_file: LICENSE.txt
  summary: Fixes some problems with Unicode text after the fact
  dev_url: https://github.com/LuminosoInsight/python-ftfy
  doc_url: http://ftfy.readthedocs.io/

extra:
  recipe-maintainers:
    - rolando
